docs/installation.md

```diff
@@ -83,147 +93,6 @@ pkg install hs-haskell-language-server
 to install it. At the moment, HLS installed this way only supports the same GHC
 version as the ports one.
 
-## Installation from source
```

Comment:
Since it doesn't include the proper suffix, not sure if we want to do that. The wrapper might pick it by accident even if the version doesn't match. But no strong opinion.

Reply:
The same is true of installing from Hackage, which we do document. That's why I'm suggesting we could document it with the same caveats.

Reply:
Fair point, will change it

Reply:
I restored the docs with notice, but tried to minimise it in order to be not redundant.